### DISCUSSION TITLE
How to exit XYBASIC back to CP/M?

### DISCUSSION THREAD

**Discussion by capnrob97** [thehighnibble/thehighnibble.github.io#123] [IMSAI8080esp replica]
I can't seem to find it documented anywhere, how do you exit XYBASIC back to CP/M prompt?

In MBASIC, you type 'system', but that doesn't work in XYBASIC.

Thanks

**Comment by villaromba** [comment]
I think it's:      Control b

**Comment by nbreeden2** [comment]
There is a link to the PDF version of the manual here:
https://www.nesssoftware.com/home/mwc/doc/xybasic/xybasic.php?Contents

Page 129 shows:
<control-B>
[Boot] As always, <control-B> exits from XYBASIC and returns to the operating system or monitor.

**Comment by capnrob97** [comment]
Thanks!
